db/migrate/20160127035010_create_fleets.rb

```diff
@@ -0,0 +1,17 @@
+class CreateFleets < ActiveRecord::Migration
+  def change
+    create_table :fleets do |t|
+      t.string :fleet_name, null: false
```

Comment:
name is a better field name here fleets.fleet_name has is redundant. Save yourself the keystrokes